pages/common/enca.md

```diff
@@ -0,0 +1,23 @@
+# enca
+
+> Detect and convert encoding of text files
+
+- detect file encoding according to your system's locale
+
+`enca {{file}}`
+
+- detect file encoding; -L option tells enca the current language; `language` is zh_CN, en_US etc.
+
+`enca -L {{language}} {{file}}`
+
+- convert file to specified encoding
+
+`enca -L {{language}} -x {{to_encoding}} {{file}}`
+
+- convert multiple files to specified encoding
+
+`enca -L {{language}} -x {{to_encoding}} *`
```

Comment:
The two examples "convert file to specified encoding" and "convert multiple files to specified encoding" could be merged into one, just say something like "convert file(s) to specified encoding" etc. `*` is a shell behavior and should not be in the examples, also "multiple files" doesn't always mean "all files (except hidden) in the current folder" :)